### ISSUE TITLE
Cannot proceed further even if accessibility granted

### ISSUE THREAD

**Issue by kartikkasat** [Caldis/Mos#709]
**Describe the bug | 问题描述** - title

**To Reproduce | 如何重现** - I tried to toggle it on and off and also restart the system still not working

**Expected behavior | 期望结果** - if its allowed, auth should proceed

**Screenshots | 相关截图**

**System Info | 系统信息:** - Mac Air M2 16GB 
 - Mouse: [e.g. Logitech MX Master] 
 - Version [e.g. v69]
 -  Application [e.g. Safari, Chrome, Whole System]

**Additional context | 额外说明**

**Comment by hellomarb** [comment]
Have you tried removing Mov from the list and re-adding it? That worked for me.

**Comment by alexleonard** [comment]
I have the same issue. I had updated my apps on Friday with Homebrew and today needed to restart. When I restarted I got asked to do the onboarding steps including toggling on Accessibility access. I've tried toggling it on and off and auth still stays on the screen asking for me to toggle the permission.

I also tried disabling the permission, restarting, and going through the process again. But I'm just stuck at this screen.

<img width="382" alt="Image" src="https://github.com/user-attachments/assets/a0b3b930-6c5a-4b56-8568-16dea4e54fd9" />

**Comment by alexleonard** [comment]
Oh I'm on 3.5.0. I just upgraded from 3.4.1

**Comment by Burarrum** [comment]
Upgraded from 3.4.1 to 3.5.0 and same issue.

**Comment by Burarrum** [comment]
I was able to fix this by removing the MOS from Accessibility all together, then going to the mos app continue thru the process.

**Comment by alexleonard** [comment]
I tried what @Burarrum suggested, although at first it didn't work. I found I needed to remove MOS from accessibility, and then I restarted the laptop and then the onboarding process completely successfully.

Good to know in case it happens again!

Great to have the app back again 👍 Thanks!

**Comment by andresestrella** [comment]
thanks @hellomarb @Burarrum
